interpreter/namerd/src/main/scala/io/buoyant/namerd/iface/NamerdInterpreterInitializer.scala

```diff
@@ -138,3 +133,18 @@ case class NamerdInterpreterConfig(
 object NamerdInterpreterConfig {
   def kind = classOf[NamerdInterpreterConfig].getCanonicalName
 }
+
+object FailureThresholdConfig {
+
+  val DefaultMinPeriod = 5000.milliseconds
+  val DefaultThreshold = 2.0
+  val DefaultWindowSize = 100
+  val DefaultCloseTimeout = 4000.milliseconds
```

Comment:
@olix0r way in here, but as discussed I think this value should be higher. closer to 60.seconds or 300.seconds